### PR TITLE
Create blog post for v1.37.0

### LINKED ISSUE #429: Blog post for v1.37

### PR DESCRIPTION
**Pull request by DNin01**
Resolves #429

### CODE REVIEW

File: content/blog/v1-37-released.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: What's new in Scratch Addons v1.37
+description: Download projects with one click and more
+date: 2024-03-30T00:00:00Z
```
Comment (DNin01) [inline content/blog/v1-37-released.md:4]: Date TBD but will probably be correct by the time we merge anyway.

File: content/blog/v1-37-released.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+author: DNin01
+---
+
+# What's new in Scratch Addons v1.37.0
```
Comment (Hans5958) [inline content/blog/v1-37-released.md:8]: Remove this heading. The title on the front matter will be used instead.

**Review by Hans5958 — CHANGES_REQUESTED** [review]
For some reason I have to fill this before I can press "Request changes." Maybe there's an issue on Refined GitHub? In any case, please take a look on my previous review.

**Review by Hans5958 — APPROVED** [review]
Any issues, @WorldLanguages?

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**